scripts/benchmark_api.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euxo pipefail
+
+cd $(dirname $0)/../api
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)